frontend/src/utils/multiLanguage.ts

```diff
@@ -0,0 +1,17 @@
+import { Language } from '~shared/types'
+
+export const convertUnicodeLocaleToLanguage = (language: Language) => {
+  switch (language) {
+    case Language.ENGLISH:
+      return 'English'
+    case Language.CHINESE:
+      return 'Chinese'
+    case Language.MALAY:
+      return 'Malay'
+    case Language.TAMIL:
+      return 'Tamil'
+    // This case should not occur
```

Comment:
comment might not be needed, since default throws an error which is expected not to occur.